gradle/projects.main.properties

```diff
@@ -38,6 +38,10 @@ nessie-server-store=servers/store
 nessie-server-store-proto=servers/store-proto
 nessie-content-generator=tools/content-generator
 nessie-protobuf-relocated=tools/protobuf-relocated
+nessie-tasks-api=tasks/api
+nessie-tasks-service=tasks/service
```

Comment:
This project's path is a parent directory of the two following ones, is that on purpose?

Also the spi module seems undeclared (see my other comment).

Reply:
oops - removed